application/src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -2496,6 +2496,10 @@ type Trip implements Node {
   gtfsId: String!
   "Global object ID provided by Relay. This value can be used to refetch this object using **node** query."
   id: ID!
+  "Mode of the trip as a netex mode (string), gtfs modes of routes are mapped into them if necessary."
+  netexMode: String
```

Comment:
I believe this isn't the NeTEx mode but applies to GTFS trips as well, doesn't it?

I think you can return the TransitMode instance rather than a String.

(I don't want to preempt the discussion.)